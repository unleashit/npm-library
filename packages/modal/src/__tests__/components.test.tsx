// import { shallow } from 'enzyme';
// import * as React from 'react';
//
// import { DefaultFooter, DefaultHeader } from '../defaults/components';
//
// describe('<Modal />', () => {
//   let wrapper: any;
//
//   it('<ModalHeader/> renders with title prop', () => {
//     wrapper = shallow(<DefaultHeader title="Login" />);
//
//     expect(wrapper.find('h3')).toHaveLength(1);
//     expect(wrapper).toMatchSnapshot();
//
//     expect(wrapper.find('h3').text()).toEqual('Login');
//   });
//
//   it('<ModalFooter/> renders with title prop', () => {
//     wrapper = shallow(<DefaultFooter title={"I'm a footer"} />);
//
//     expect(wrapper.text()).toEqual("I'm a footer");
//   });
// });
