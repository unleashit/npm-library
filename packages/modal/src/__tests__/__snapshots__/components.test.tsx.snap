// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> <ModalHeader/> renders with title prop 1`] = `
<Fragment>
  <h3>
    Login
  </h3>
</Fragment>
`;
